### PR TITLE
Updated job requirements, added DESIRED_Sites/UNDESIRED_Sites.

### PR DESCRIPTION
**Pull request by rynge**

### CODE REVIEW

File: start/resources/requirements.md [hunk 1]
```diff
@@ -74,44 +71,36 @@ Below is a list of common attributes that you can include in your submit file `r
...
+To run your jobs on a list of specific execution sites, or avoid a set of 
+sites, use the `+DESIRED_Sites`/`+UNDESIRED_Sites` attributes in your job
+submit file. These attributes should only be used as a last resort. For
+example, it is much better to use feature attributes (see above) to make
+your job go to nodes matching what you really require, than to broadly
+allow/block whole sites.
 
-- **HAS_TCSH** - Boolean specifying if the node has /bin/tcsh
+To avoid certain sites, first find the site names. You can find a 
+current list by quering the pool:
```
Comment (ChristinaLK) [inline start/resources/requirements.md:95]:
```suggestion
current list by querying the pool:
```

File: start/resources/requirements.md [hunk 2]
```diff
@@ -74,44 +71,36 @@ Below is a list of common attributes that you can include in your submit file `r
...
 - **CUDACapability** - For GPU jobs, specifies the CUDA compute capability.
 
-- **HAS_SQUID** - Boolean specifying the need for access to SQUID fileshare,
-  set to `true` if your job has the _OSG_SQUID_LOCATION_ environment
-  variable set
+# Specifying Sites / Avoiding Sites
+
+To run your jobs on a list of specific execution sites, or avoid a set of 
+sites, use the `+DESIRED_Sites`/`+UNDESIRED_Sites` attributes in your job
+submit file. These attributes should only be used as a last resort. For
```
Comment (ChristinaLK) [inline start/resources/requirements.md:89]:
```suggestion
submit file. **These attributes should only be used as a last resort.** For
```

File: start/resources/requirements.md [hunk 3]
```diff
@@ -74,44 +71,36 @@ Below is a list of common attributes that you can include in your submit file `r
...
+example, it is much better to use feature attributes (see above) to make
+your job go to nodes matching what you really require, than to broadly
+allow/block whole sites.
```
Comment (ChristinaLK) [inline start/resources/requirements.md:92]:
```suggestion
allow/block whole sites. We encourage you to contact the facilitation team before taking this action, to make sure it is right for you. 
```

File: start/resources/requirements.md [hunk 4]
```diff
@@ -74,44 +71,36 @@ Below is a list of common attributes that you can include in your submit file `r
...
 
-- **HAS_TCSH** - Boolean specifying if the node has /bin/tcsh
+To avoid certain sites, first find the site names. You can find a 
+current list by quering the pool:
 
-- **HAS_XRDCP** - Boolean specifying if the node has _xrdcp_ in the default path
+    condor_status -af GLIDEIN_Site | sort -u
 
-- **HAS_TIMEOUT** - Boolean specifying if the node has _timeout_ in the default path
+In your submit file, add a comma separated list of those site like:
```
Comment (ChristinaLK) [inline start/resources/requirements.md:99]:
```suggestion
In your submit file, add a comma separated list of sites like:
```

File: start/resources/requirements.md [hunk 5]
```diff
@@ -74,44 +71,36 @@ Below is a list of common attributes that you can include in your submit file `r
...
   /proc/cpuinfo
-- **CVMFS_oasis_opensciencegrid_org_REVISION** - Attribute specifying
-  the need to access specific oasis /cvmfs file system repositories. See the
-  example above for determining the appropriate REVISION number when using
-  this `requirements` attribute.
+
+- **HAS_CVMFS_oasis_opensciencegrid_org** - Attribute specifying
+  the need to access specific oasis /cvmfs file system repositories.
 
 - **CUDACapability** - For GPU jobs, specifies the CUDA compute capability.
```
Comment (ChristinaLK) [inline start/resources/requirements.md:83]:
```suggestion
- **CUDACapability** - For GPU jobs, specifies the CUDA compute capability.
See our [GPU guide](5000653025) for more details. 
```

File: start/resources/requirements.md [hunk 6]
```diff
@@ -1,65 +1,62 @@
...
-	undefined
+	$ condor_status -pool cm-1.ospool.osg-htc.org -af OSGVO_OS_STRING | sort -u
+    RHEL 7
+    RHEL 8
 
-This means that we can specify an OS version of `6`, `7`, `8`. Alternatively
-you will find many attrubites will take the boolean values `true` or `false`
-as shown in the HAS_MODULES example above.
+This means that we can specify an OS version of `RHEL 7` or `RHEL 8`. Alternatively
+you will find many attrubites will take the boolean values `true` or `false`.
```
Comment (ChristinaLK) [inline start/resources/requirements.md:59]:
```suggestion
you will find many attributes will take the boolean values `true` or `false`.
```

**Review by ChristinaLK — COMMENTED** [review]
a few tiny typos and some emphasis/links to other guides added

File: start/resources/requirements.md (hunk 5 above)
Comment (rynge) [reply start/resources/requirements.md:83]: Added this as a commit as github would not let me direct commit it.

File: start/resources/requirements.md [hunk 7]
```diff
@@ -1,65 +1,62 @@
...
 
-	$ /usr/bin/attr -g revision /cvmfs/oasis.opensciencegrid.org
-
-### Additional Feature-Specific Attributes
 There are many attributes that you can use with `requirements`. To see what values
 you can specify for a given attribute you can run the following command while
 connected to your login node:
 
-	$ condor_status -pool flock.opensciencegrid.org -af {ATTR_NAME} | sort -u
+	$ condor_status -pool cm-1.ospool.osg-htc.org -af {ATTR_NAME} | sort -u
```
Comment (ChristinaLK) [inline start/resources/requirements.md:50]: do we even need the pool options? 
```suggestion
	$ condor_status -af {ATTR_NAME} | sort -u
```

File: start/resources/requirements.md [hunk 8]
```diff
@@ -1,65 +1,62 @@
...
 	
-For example, to see what values you can specify for the OSGVO_OS_VERSION attribute run:
+For example, to see what values you can specify for the OSGVO_OS_STRING attribute run:
 	
-	$ condor_status -pool flock.opensciencegrid.org -af OSGVO_OS_VERSION | sort -u
-	6
-	7
-	8
-	undefined
+	$ condor_status -pool cm-1.ospool.osg-htc.org -af OSGVO_OS_STRING | sort -u
```
Comment (ChristinaLK) [inline start/resources/requirements.md:54]:
```suggestion
	$ condor_status -af OSGVO_OS_STRING | sort -u
```

File: start/resources/requirements.md [hunk 9]
```diff
@@ -74,44 +71,37 @@ Below is a list of common attributes that you can include in your submit file `r
...
 
-- **HAS_TIMEOUT** - Boolean specifying if the node has _timeout_ in the default path
+In your submit file, add a comma separated list of sites like:
 
-- **HAS_R** - Boolean specifying if the node has R
+    +UNDESIRED_Sites = "ISI,SU-ITS"
 
-- **HAS_NUMPY** - Boolean specifying if the node has Python Numpy
+Those sites will now be exluded from the set of sites your job can
+run at.
```
Comment (ChristinaLK) [inline start/resources/requirements.md:105]: also say something about `+DesiredSites` or omit it above.

**Review by ChristinaLK — APPROVED** [review]

File: start/resources/requirements.md (hunk 9 above)
Comment (rynge) [reply start/resources/requirements.md:105]: Addressed in new commit

**Pull request merged**